Readme.md

```diff
@@ -106,6 +107,10 @@ let package = Package(
 
 Note that as of Moya 10, SPM only works with Swift 4 toolchain and greater.
 
+### Accio
+
+Accio is based on Swift Package Manager (with the addition of iOS/macOS/tvOS/watchOS support) and therefore integration of Moya is exactly the same as described above. Once your `Package.swift` file is configured, run `accio update`.
```

Comment:
Just a sidenote, I tried to google "Accio Dependency GitHub" and it found not only a same-named project (which is fine usually), but it's also a dependency manager. Might make it a bit hard to find :) 

I'd add a link on the word "Accio" in the beginning of the sentence and change it to a

Accio is a dependency manager based on Swift Package Manager.

I'd get rid of the addition of support thing because SwiftPM does support these platforms in v5 AFAIK: 

![image](https://user-images.githubusercontent.com/605076/55645059-367d8f80-57e0-11e9-8dcb-8c4a4d88f618.png)

Reply:
Interesting, seems like someone had the same name idea, I didn‘t optimize for SEO at all but I hope the fact that the other one is for C++ will make them still distinguishable.

Regarding platforms support, I disagree. SwiftPM does NOT support all Apple platforms, it only now has a way of specifying what the minimum required version of a platform is - that‘s a feature of the manifest file though, not of the tool itself. I think to most it is clear that SwiftPM doesn‘t support iOS, otherwise the community would have opted to use it a long time ago.

Regarding linking to Accio, no problem, I can add a link.